archivist/events.py

```diff
@@ -69,15 +133,46 @@ def __query(props, attrs, asset_attrs):
     def count(
         self, *, asset_id=ASSETS_WILDCARD, props=None, attrs=None, asset_attrs=None
     ):
-        """docstring"""
+        """Count events.
+
+        Counts number of events that match criteria.
+
+        Args:
+            asset_id (str): optional asset identity e.g. assets/xxxxxxxxxxxxxxxxxxxxxxxxxx
+            props (dict): optional properties e.g. {"confirmation_status": "CONFIRMED" }
+            attrs (dict): optional attributes e.g. {"arc_display_type": "door" }
+            asset_attrs (dict): optional asset_attributes e.g. {"arc_display_type": "door" }
+
+        Returns:
+            integer count of assets.
+
+        """
+
         return self._archivist.count(
             SEP.join((ASSETS_SUBPATH, asset_id, EVENTS_LABEL)),
             query=self.__query(props, attrs, asset_attrs),
         )
 
-    def wait_for_confirmed(self, *, asset_id=ASSETS_WILDCARD, props=None, attrs=None):
-        """docstring"""
-        return wait_for_confirmed(self, asset_id=asset_id, props=props, attrs=attrs)
+    def wait_for_confirmed(
+        self, *, asset_id=ASSETS_WILDCARD, props=None, attrs=None, asset_attrs=None
+    ):
+        """Wait for events to be confirmed.
+
+        Waits for all events that match criteria to be confirmed.
+
+        Args:
+            asset_id (str): optional asset identity e.g. assets/xxxxxxxxxxxxxxxxxxxxxxxxxx
+            props (dict): e.g. {"tracked": "TRACKED" }
```

Comment:
‘Door’ is not a reasonable event display type, and doesn’t help to disambiguate event_attributes from asset_attributes. Leave the asset example as ‘door’ but change this event one to ‘open’